versions.props

```diff
@@ -27,8 +27,9 @@ com.palantir.config.crypto:encrypted-config-value-module = 2.1.0
 com.palantir.conjure:conjure = 4.10.1
 com.palantir.conjure.java.api:* = 2.9.0
 com.palantir.conjure.java:* = 5.11.2
-com.palantir.dialogue:* = 1.2.0
-com.palantir.conjure.java.runtime:* = 4.70.1
+com.palantir.dialogue:* = 1.11.4
+com.palantir.conjure.java.runtime:* = 5.3.0
+com.github.tomakehurst:wiremock-standalone = 2.26.3
```

Comment:
We already have this version slightly higher, this library is used for some tests.